x/supplier/keeper/query_supplier.go

```diff
@@ -52,7 +53,8 @@ func (k Keeper) Supplier(goCtx context.Context, req *types.QueryGetSupplierReque
 		req.Address,
 	)
 	if !found {
-		return nil, status.Error(codes.NotFound, "not found")
+		msg := fmt.Sprintf("supplier with address %q", req.GetAddress())
```

Comment:
Going to think about how to iterate on this moving forward but just approving for now.